tests/unittest/tools/test_layer_wise_benchmarks.py

```
@@ -339,6 +339,42 @@ def test_qwen3_next_gen_tep(llm_root, world_size):
     )
 
 
+# Kimi K3's MXFP4 routed experts and KDA kernels require SM100+.
+@skip_pre_blackwell
+@pytest.mark.parametrize("world_size", [4])
+def test_kimi_k3_gen_dep(llm_root, world_size):
+    if torch.cuda.device_count() < world_size:
+        pytest.skip(f"needs {world_size:d} GPUs to run this test")
+    model_root = llm_models_root(check=True)
+    profile_dir = f"profiles/test_kimi_k3_gen_dep_{world_size}"
+    check_call(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)